### DISCUSSION TITLE
Homeassistant OS VM - virtual network interface get's wrong IP on install

### DISCUSSION THREAD

**Discussion by simon041988** [tteck/Proxmox#1339] [Help Needed]
Hi there,

first of all thank you so much for these great scripts that make working with Proxmox a breeze!
Thank you for all the work you invested and still do!

I used many of your scripts already, mostly containers, and now I tried to install the homeassistant OS VM.
The installation went smooth and the VM started with no obvious issues. 

But when I tried to look up the ip address of the VM in the dashboard, I recognized, that the ip it received uses a completely different ip address range than I use in my network. 
![Bildschirm­foto 2023-04-16 um 20 07 20](https://user-images.githubusercontent.com/42072768/232332800-caa50763-96a9-4172-97b6-9076514180c4.png)

My network is at 192.168.x.x 

The "nmcli con show" command doesn't list any network interfaces. 

I am running Proxmox 7.4-3 on a supermicro server with 128 GB RAM, 16 cores, and 8 network ports installed.   

I didn't experience the behavior with my other VM's on the same server. 

I'm pretty sure the problem is sitting in front of the screen ;-) But can somebody assist me in seeing able to access the home assistant dashboard? 

Thank you very much in advance!

Simon

**Comment by tteck** [comment]
You're looking at the internal Docker network.
Go to the VM console to find the HAOS IP

![Screenshot 2023-04-16 2 34 54 PM](https://user-images.githubusercontent.com/21342670/232334498-35847020-dea2-41ab-bf51-287dd99df28d.png)

**Reply by tteck** [reply]
Or, the Summary tab

![Screenshot 2023-04-16 2 40 28 PM](https://user-images.githubusercontent.com/21342670/232334637-30524a7c-f166-49f6-86db-9295b3c236d8.png)

**Reply by simon041988** [reply]
Thank you for your reply! The IP in the summary is just the same
![Bildschirm­foto 2023-04-16 um 21 10 04](https://user-images.githubusercontent.com/42072768/232336425-0af0b560-42c2-420a-ad09-78414be18717.png)

And when I access the console it looks like this:

![Bildschirm­foto 2023-04-16 um 21 11 11](https://user-images.githubusercontent.com/42072768/232336477-951c2d95-dec2-479c-a7f4-a0678a387401.png)

Anny other suggestion? Thank you in advance!!

**Reply by tteck** [reply]
Then it didn't receive an IP from your DHCP server (router). 
Are you using default settings?

**Reply by simon041988** [reply]
Yes - I am using default settings (besides RAM). Tried different install types - same result. I am just confused because I never ran into this problem with my other VM's or containers....

**Reply by tteck** [reply]
Are you out of assignable IP addresses?
Can you create any other LXC's?

**Reply by simon041988** [reply]
That shouldn't be a problem - but good point. I increased the IP range of the DHCP server anyway and rebooted the VM - the problem persists. 
I can create other LXC's without problems. I also tried to reboot Proxmox.

**Comment by simon041988** [comment]
Just tried to use Version 10.0 RC of Homeassistant instead V9.X. No problem here... For sure no issue with your script. Thank you very much for your work and help. You can mark the topic as resolved. :-)
